.github/workflows/check.yml

```diff
@@ -95,7 +95,7 @@ jobs:
         run: cargo build --features shuttle
```

Comment:
You should build this with `--release` too, right?  I mean, we wouldn't be building or testing with debug and shuttle, but that's fine.

Reply:
I tried to cheat here and use this command to cover debug builds (if shuttle test fails, it is very hard to debug shuttle tests in release build) as well. I can create another CI step to keep it clean - after taking another look, I no longer like this approach